openshift/worker.yaml

```diff
@@ -245,16 +253,36 @@ parameters:
   name: NAMESPACE
   required: true
   value: myproject
+- description: Number of Celery Workers to create
+  displayName: Celery Worker Replicas
+  name: CELERY_WORKER_REPLICAS
+  required: true
+  value: '1'
+- description: Initial amount of memory the Flower container will request.
+  displayName: Celery Worker Memory Request
+  name: CELERY_WORKER_MEMORY_REQUEST
+  required: true
+  value: 1Gi
 - description: Maximum amount of memory the Flower container can use.
-  displayName: Memory Limit
+  displayName: Celery Worker Memory Limit
   name: CELERY_WORKER_MEMORY_LIMIT
   required: true
   value: 1Gi
```

Comment:
Let's get the worker limit to the max 2GB

Reply:
Updated